code/__HELPERS/lighting.dm

```diff
@@ -4,6 +4,11 @@
 	// Consider removing it someday? (I wonder if we made emissives blend right we could make alpha actually matter. dreams man, dreams)
 	var/mutable_appearance/appearance = mutable_appearance(icon, icon_state, layer, offset_spokesman, EMISSIVE_PLANE, 255, appearance_flags | EMISSIVE_APPEARANCE_FLAGS, offset_const)
 	appearance.color = GLOB.emissive_color
+
+	if(icon_state && !icon_exists(icon, icon_state, FALSE))
+		//We were meant to add... something, but the icon state didn't exist. (Scream set to False so we can have a custom stack_trace)
```

Comment:
The comment explaining why `FALSE` should be before/on the line itself

Reply:
Split it up a bit, so the actual test is described the line above, and the `FALSE` explanation is on the line itself 👍